src/components/Login/EnterPhonePanel.tsx

```diff
@@ -0,0 +1,69 @@
+import { useState } from 'react'
+
+import Form from '@/components/Login/LoginForm'
+import Button from '@/components/UI/button'
+import { SelectOption } from '@/types/types'
+
+export interface EnterPhonePanelInterface {
+  onClick: (phoneNumber: string) => void
+}
+
+const EnterPhonePanel = ({ onClick }: EnterPhonePanelInterface) => {
+  const [displaynumber, setdisplaynumber] = useState('')
+  const [countrycode, setcountrycode] = useState('')
+
+  const countryCodeSelectOptions: SelectOption[] = [
+    { value: '+61', label: 'Australia (+61)' }
+  ]
+
+  function handleSubmit(event: React.ChangeEvent<HTMLFormElement>) {
+    onClick(countrycode + displaynumber)
+    event.preventDefault()
+    event.target.reset()
+  }
+
+  function formFilled() {
+    return displaynumber !== ''
+  }
+
+  return (
+    <div className='flex w-1/4 flex-auto flex-col'>
```

Comment:
There's a bug with the input field width
![image](https://user-images.githubusercontent.com/34858205/213120747-b73dbb5a-501f-4a16-b396-17f29386d917.png)